change/@fluentui-react-charting-e13576b2-4b54-41bc-a96f-c30b8ada3470.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Adjust horizontal bar chart styles as per design",
+  "packageName": "@fluentui/react-charting",
+  "email": "kumarkshitij@microsoft.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
Get the end result reviewed from the designer

Reply:
ok